### ISSUE TITLE
Assertion when using remote object as argument in method on other remote object

### ISSUE THREAD

**Issue by pontusn** [MicrosoftEdge/WebViewFeedback#148]
Our legacy API returns complex objects. To reduce latency overhead we added new helper objects with methods that convert legacy objects to JSON. 

When calling this helper from JS with any other remote object as argument we consistently see this assertion in the developer console:

```
                    _updateSequenceNumberForProxyOptions(options, sequenceId) {
                        console.assert(options && options.remoteSequenceId <= sequenceId);
                        options.remoteSequenceId = sequenceId;
                    }
```

**Comment by david-risney** [comment]
Another great AddRemoteObject bug. Thank you for finding these! Please let us know what other AddRemoteObject (or otherwise) issues you find.

**Comment by pontusn** [comment]
You are welcome. I work as a freelance developer. My current client has a huge legacy MFC/ATL/Win32-application with some 60k users that currently embeds IE11. My current job is to prepare for migration to WebView2 ASAP. This application is great platform for finding problems. It has components from all eras since mid nineties and uses embedded webbrowser for all customized interactions. And once we flip the switch there will be many end-users.

**Comment by champnic** [comment]
This should be fixed in Edge 84.0.508.0+. Thanks!
